Chess_Board/src/board.py

```diff
@@ -67,8 +67,10 @@ def check_promotion(self, piece, final):
     def castling(self, initial, final):
         return abs(initial.col - final.col) == 2
 
+    # this function is suppose  to set all piece to false en_passant except pawn  
     def set_true_en_passant(self, piece):
         
+        
```

Comment:
@Ahmedniz  why did you add this blank line?

Reply:
@Salman-18 Ok let me fix that